3.optimizing_CP/scripts/1.assess_best_pipeline.py

```diff
@@ -0,0 +1,314 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+import argparse
+import pathlib
+import pprint
+import sys
+
+import optuna
+import pandas as pd
+import torch
+
+sys.path.append("../CellProfiler_optuna_utils/")
+from optuna_profiling_utils import (
+    adjust_cpipe_file_LAP,
+    adjust_cpipe_file_overlap,
+    adjust_cpipe_file_save_tracked_object_images,
+    extract_pixel_number_for_overlap_tracking,
+    extract_single_time_cell_tracking_entropy,
+    harmonic_mean,
+    loss_function_from_CP_features,
+    loss_function_MSE,
+    remove_trial_intermediate_files,
+    retrieve_cell_count,
+    run_CytoTable,
+    run_pyctominer_annotation,
+)
```

Comment:
Can you just import all the utils instead of defining each of them? This might just be a preference, but I am curious :)

Reply:
I could, but it is not best practice